project/source/spawn_test/SpawnTest.cpp

```diff
@@ -68,7 +68,7 @@ namespace nc::sample
         staticCubeSpawner->SetPrefab(prefab::Resource::CubeRed);
         staticCubeSpawner->Spawn(5);
 
-        // Fixed interval spawner for moving cubes
+        //Fixed interval spawner for moving cubes
```

Comment:
Why is this changed? I prefer one space, and the others in this file are one space.

Reply:
no idea